python/plugins/processing/algs/grass7/TODO

```diff
@@ -1,9 +1,70 @@
-* http://trac.osgeo.org/grass/wiki/Grass7/NewFeatures#Replacedandremovedmodules
-* TODO: Merged modules
-        r.average: merged into G7:r.statistics, G7:r.statistics2, G7:r.statistics3
-        r.bilinear merged into G7:r.resamp.interp
-        r.median: merged into G7:r.statistics, G7:r.statistics2, G7:r.statistics3
-        r.sum: merged into G7:r.statistics, G7:r.statistics2, G7:r.statistics3
-* http://trac.osgeo.org/grass/wiki/Grass7/NewFeatures#Renamedmodules
+TODO List for GRASS7 algorithms support into QGIS Processing
+
+Unit tests
```

Comment:
Just a small thing, if you rename this file to TODO.md it will be formatted automatically, especially since the syntax seems to be markdown already.